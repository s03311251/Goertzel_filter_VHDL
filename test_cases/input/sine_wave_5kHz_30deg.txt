2FFF
30DC
31B5
3289
3358
3423
34E8
35A8
3663
3718
37C7
3870
3913
39B0
3A46
3AD5
3B5E
3BE0
3C5B
3CCE
3D3B
3DA0
3DFD
3E53
3EA1
3EE8
3F27
3F5D
3F8C
3FB3
3FD2
3FE9
3FF8
3FFF
3FFD
3FF4
3FE2
3FC9
3FA7
3F7E
3F4C
3F13
3ED1
3E88
3E37
3DDF
3D7F
3D18
3CA9
3C33
3BB6
3B31
3AA6
3A15
397C
38DD
3838
378D
36DC
3625
3569
34A7
33E0
3313
3243
316D
3093
2FB5
2ED3
2DED
2D03
2C17
2B27
2A34
293F
2848
274E
2653
2556
2457
2358
2257
2157
2055
1F54
1E53
1D52
1C52
1B53
1A55
1958
185E
1765
166E
157A
1488
1399
12AE
11C5
10E1
