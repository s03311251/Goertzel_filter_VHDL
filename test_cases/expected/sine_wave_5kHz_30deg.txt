FFFEB680
FFFEC8E0
